src/query/QueryManager.jl

```diff
@@ -39,7 +38,7 @@ function match_group_aggregate(transaction::AbstractCoreTransaction, query::Stri
     return result
 end
 
-function insert(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
+function insert(transaction::AbstractCoreTransaction, query::AbstractString, options = Proto.Options())
     db_result = stream(transaction, QueryManagerRequestBuilder.insert_req(query, options))
     if db_result !== nothing
```

Comment:
I still feel it's cleaner to have it exit early when `db_result` is `nothing`.
Also, having the `map` creating an extra vector allocation can be a performance issue, so that should definitely be changed, to be like the above cases that have `ConceptMapGroup` and `NumericGroup` as part of the array comprehensions.